### PR TITLE
Implement Navbar for Desktop

### PR DESCRIPTION
**Pull request by ivan-unfolds**
* Deleted isotope JS file and links to it.
* Pruned main.js.
* Finished CAST side navbar.

Relates #112

### CODE REVIEW

File: source/layouts/_main_nav.html.haml [hunk 1]
```diff
@@ -1,22 +1,21 @@
...
+
+          %ul#navigation.nav.navbar-nav.ma0.pa0.flex.flex-column.items-center.flex-row-l
+            - liClass = "w-90 br2 w-auto-l tc-l ml2-l white hover-light-red hover-bg-white transition-300 transition-all"
+            - data.navbar.headers.each_with_index do |header, i|
+              %li{ class: "#{liClass}" ++ (i == 0 ? " active" : "") }
+                = link_to header.title, header.href, class: "db pa3 f6 br2  pa3-l color-inherit"
+
+          .social-links.dn.flex-l.flex-row-l.bg-blue-green.items-center-l.pl3-l.pr3-l
+            - data.navbar.icons.each do |icon|
+              %a{ href: icon.href, target: "_blank", class: "white pl3-l hover-light-red"}
```
Comment (andrewMacmurray) [inline source/layouts/_main_nav.html.haml]: add these two here to make the smooth transition: `transition-300 transition-all` ✨

**Review by andrewMacmurray — APPROVED** [review]
Looks good, add those classes and it's 👌

File: source/layouts/_main_nav.html.haml (hunk 1 above)
Comment (ivan-unfolds) [reply source/layouts/_main_nav.html.haml]: @andrewMacmurray Ready for your consideration.

**Comment by andrewMacmurray** [comment]
👌 👍

**Pull request merged**